TuneUp/TuneUp.csproj

```diff
@@ -64,34 +62,31 @@
     </Page>
   </ItemGroup>
   <ItemGroup>
-    <PackageReference Include="DynamoVisualProgramming.Core">
-      <Version>2.12.0.5650</Version>
-      <ExcludeAssets>runtime</ExcludeAssets>
-      <IncludeAssets>compile; build; native; contentfiles; analyzers</IncludeAssets>
-    </PackageReference>
-    <PackageReference Include="DynamoVisualProgramming.DynamoServices">
-      <Version>2.12.0.5650</Version>
-      <ExcludeAssets>runtime</ExcludeAssets>
-      <IncludeAssets>compile; build; native; contentfiles; analyzers</IncludeAssets>
-    </PackageReference>
-    <PackageReference Include="DynamoVisualProgramming.WpfUILibrary">
-      <Version>2.12.0.5650</Version>
-      <ExcludeAssets>runtime</ExcludeAssets>
-      <IncludeAssets>compile; build; native; contentfiles; analyzers</IncludeAssets>
-    </PackageReference>
+	  <PackageReference Include="DynamoVisualProgramming.Core" Version="3.2.1.5366">
+		  <ExcludeAssets>runtime</ExcludeAssets>
+		  <IncludeAssets>compile; build; native; contentfiles; analyzers</IncludeAssets>
```

Comment:
Any reason to explicitly add which assets to include here? Other projects don't do it.

Reply:
This one is in `master`. Given the brief I thought of keeping the same logic .
Will correct.